### ISSUE TITLE
Unable to install some scripts in version 1.4.0

### ISSUE THREAD

**Issue by oscarotero** [denoland/deno#7455]
Hi, I just upgraded to the latest version and am having some problems installing libraries.

I'm working on this static site generator: https://deno.land/x/lume@v0.2.2

I've a downgrade to `1.3.3` and the library is installed and work fine:

```sh
$ deno upgrade --version 1.3.3
$ deno install -f --unstable --allow-read --allow-write --allow-net https://deno.land/x/lume@v0.2.2/cli.js
$ lume

#Site build successfully
```

But after upgrade to `1.4.0`:

```sh
$ deno upgrade --version 1.4.0
$ deno install -f --unstable --allow-read --allow-write --allow-net https://deno.land/x/lume@v0.2.2/cli.js
$ lume

error: Uncaught TypeError: Cannot read property 'default' of undefined
export default __exp["default"];
                    ^
    at file:///Users/oscarotero/.deno/bin/lume.js:47172:21
```

The library works fine if it's executed locally, without install it (with `deno run --unstable --allow-read --allow-write --allow-net cli.js`), but it's failing on install. And because in the new version, the code is bundled before installation, I cannot know what is the source's filename and line that is failing.

**Comment by lucacasonato** [comment]
This issue seems to be caused by the bundling:

```shell
~ ❯❯❯ deno bundle --unstable https://deno.land/x/lume@v0.2.2/cli.js > luma.js
Bundle https://deno.land/x/lume@v0.2.2/cli.js
~ ❯❯❯ deno run --unstable luma.js 
error: Uncaught TypeError: Cannot read property 'default' of undefined
export default __exp["default"];
                    ^
    at file:///home/lucacasonato/luma.js:47172:21
```

**Comment by kitsonk** [comment]
@lucacasonato no, see: #4499 and #4549. #7225 should address this though.

**Comment by kitsonk** [comment]
I realise having install do a bundle sounds great, but in retrospect, there are quite a few scenarios we can't/don't support with bundling at the moment.  I am not sure it is viable, or it needs to be an option on `deno install` instead of it being forced.

**Comment by oscarotero** [comment]
Having an option would be great. Like `deno install --bundle {url}`

**Comment by bartlomieju** [comment]
This problem must have been caused by #5276. I expected it might have broken some code but we decided to land it anyway. I guess we'll have to revert this functionality until bundling is finished.

**Comment by oscarotero** [comment]
This problem has been fixed in 1.4.1, so I'm closing this.
